### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Configure CORS Settings _ScreenReader:As soon as error message appears on the screen, screenreader does not announce the error message. 

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2097]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS3.3.2;#NVDA;#JAWS; #Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
NVDA 2019.2
JAWS 2019
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and Select "Configure CORS Settings" and hit enter
5.Open NVDA and check whether it reads the error message as soon as appear on the screen.

**​Actual:​**
As soon as error message appears on the screen, screenreader does not announce the error message. 
**Same issue is observed in JAWS also.**

**Expected**:
Screenreader should read the error message as soon as appears on the screen.

**UserImpact:**
Screen reader users unable to get the error message.

**Comment by craxal** [comment]
Element just needs an `alert` role.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT  Verified on version 1.12.0 and we are not getting the error message while fill the form.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
